array-methods/for-each.ts

```diff
@@ -0,0 +1,11 @@
+const values = [10, 1, 22, 23, 41, 5, 18, 7, 80, 9];
+console.log('In order:');
+values.forEach((element) => {
+  console.log(element);
+});
+console.log('Reverse order:');
+
+values.forEach((element) => {
+  const index = values.indexOf(element);
+  console.log(values[9 - index]);
+});
```

Comment:
You want to avoid using hard coded values like `9`, because what if our array changes? You want this to always work dynamically with whatever array, so it would be better to utilize the `length` of the array

```
values.forEach((n, index) => console.log(values[values.length - 1 - index]));
```